README.md

```diff
@@ -17,7 +17,7 @@ Probably the most well known textbook on machine learning. Recommended for mathe
 
 * Marina also has a [list of books on probability](http://www.stat.washington.edu/courses/stat391/spring15/books.html), for STAT 391 students. 
 
-* UW faculty member [Jake VanderPlas](http://staff.washington.edu/jakevdp/) is working on an O'Reilly book "Python Data Science Handbook". Some previews are [posted](https://www.oreilly.com/people/89c9c-jake-vanderplas).
+* UW faculty member [Jake VanderPlas](http://staff.washington.edu/jakevdp/) is the author of O'Reilly's "Python Data Science Handbook". All notebooks and resources used in the book are [posted](https://github.com/jakevdp/PythonDataScienceHandbook/tree/master/notebooks).
 
 ## Software
 
```

Comment:
I can't comment on L24 for some reason, but it might be good to revise...I feel like Python is the predominant language for machine learning / data science, with R as a language for statistical modeling. Besides, we don't teach anything in R.